doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -15,12 +15,12 @@ Use these values to see whether your application has enough space for running on
 How to read the tables
 **********************
 
-The memory requirement tables list flash memory and RAM requirements for samples that were built using the available :ref:`nrfxlib:ot_libs`.
+The memory requirement tables list flash memory and RAM requirements for samples that were built from sources.
 
 The values change depending on the sample, device type, and hardware platform.
 Moreover, take into account the following considerations:
 
-* All samples were compiled using the default :file:`prj.conf`, the overlay :file:`overlay-ot-defaults`, and images from :ref:`nrfxlib:ot_libs`.
+* All samples were compiled using the default :file:`prj.conf` plus the proper :kconfig:`OPENTHREAD_NORDIC_LIBRARY` and :kconfig:`OPENTHREAD_THREAD_VERSION` choices.
```

Comment:
What does "proper" mean here?

Reply:
We provide memory usage information for different configurations: Master, FTD, MTD, plus Thread 1.1 and Thread 1.2. So those two configurations options are selected accordingly.

Reply:
I think "corresponding" works better instead of "proper".